source/MaterialXView/Viewer.cpp

```diff
@@ -2395,7 +2343,7 @@ mx::MaterialPtr Viewer::getEnvironmentMaterial()
 {
     if (!_envMaterial)
     {
-        mx::FilePath envFilename = _searchPath.find(mx::FilePath("resources/Lights/envmap_shader.mtlx"));
+        mx::FilePath envFilename = _searchPath.find(mx::FilePath("resources/Materials/TestSuite/nprlib/environment_map.mtlx"));
```

Comment:
IIRC MaterialXViewer usually does not refer to resources in TestSuite. Is envmap being relocated to npr due to dependency on viewdirection node?

Reply:
That's a good point, and we could indeed keep this resource in the "Lights" folder if that's more consistent with existing usage.  Originally, I moved it to the TestSuite so that it would be validated in MaterialXTest, but we have other `viewdirection` examples in that folder that can serve in this role, and the environment map shader doesn't strictly need to be there.